### DISCUSSION TITLE
GPU tensorlflow DLClive error

### DISCUSSION THREAD

**Discussion by MilanNedoma** [bonsai-rx/bonsai#1598] [Q&A]
Hi!

I have managed to put together a pipeline for data gathering using bonsai with DLClive. A week ago, everything was working smoothly, but now, when I try to run the workflow, it displays multiple error messages. First, it was asking for a lower version of the files that have already been in the CUDA toolkit bin file (namely: cublas64_11.dll, cublasLt64_11.dll, cufft64_10.dll & cusparse64_11.dll). I have checked and there have already been these present, each with a higher version. After exchanging them for the ones mentioned above, now the following message pops up. I have again managed to find the cusolver.dll in the folder, therefore, I have no idea what to do at this point. As it can be seen, it does not utilize the GPU properly. From my side, there have been no updates to either the PC, Bonsai or the DLC. Could I ask for help in this regard? 
Please see the attached workflow and the screenshot for reference.

Thank you very much in advance.

![BonsaiDLClive_Problem](https://github.com/bonsai-rx/bonsai/assets/140824061/f73a8ec8-b1d2-4dc7-a629-95456ce87e10)
[Workflow.txt](https://github.com/bonsai-rx/bonsai/files/13176290/Workflow.txt)

**Comment by chiyu1203** [comment]
It looks like you are missing that library. Have you tried to download "cusolver64_11.dll" from somewhere? Also could you describe a little bit more about your python, deeplabcut and bonsai environment? I had kept an installation diary before. You can have a look [here](https://github.com/bonsai-rx/deeplabcut/issues/18)
